signer/src/transaction_signer.rs

```diff
@@ -160,6 +168,35 @@ struct AcceptedSigHash {
     public_key: PublicKeyXOnly,
 }
 
+/// An enum identifying requests for which we can sign for on stacks only once
+/// per tenure.
+#[derive(Clone, Debug, Hash, PartialEq, Eq)]
+pub enum StacksSignRequestId {
```

Comment:
Spoke about this offline and think that we should add variants for the other stacks sign requests that we have. It doesn't hurt to do so.